packages/keeper-bots/src/converter-bot/providers/uniswap.ts

```diff
@@ -103,14 +104,16 @@ class UniswapProvider extends SwapProvider {
 
         trade = {
           inputToken: {
-            amount: BigInt(
-              new Fraction(inputCurrency.numerator, inputCurrency.denominator).toFixed(0, { groupSeparator: "" }),
+            amount: new Fraction(
+              BigInt(inputCurrency.numerator.toString()),
+              BigInt(inputCurrency.denominator.toString()),
             ),
             address: (inputCurrency?.currency as Token).address as Address,
           },
           outputToken: {
-            amount: BigInt(
-              new Fraction(outputCurrency.numerator, outputCurrency.denominator).toFixed(0, { groupSeparator: "" }),
+            amount: new Fraction(
+              BigInt(outputCurrency.numerator.toString()),
+              BigInt(outputCurrency.denominator.toString()),
```

Comment:
Aren't `numerator` and `denominator` already bigints?

Reply:
Uniswap uses JSBI